src/schematools/types.py

```diff
@@ -1093,6 +1093,12 @@ def required(self) -> bool:
 
     @property
     def type(self) -> str:
+        """Returns the type of this field.
```

Comment:
Maybe you can also point to the documentation at: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#data-types

Reply:
Done, please check again.